docs: Getting started section is wrong

Hi! I've tried to create a new project using `gpui-component` but unfortunately after following the steps the described on the [Getting started](https://longbridge.github.io/gpui-component/docs/getting-started) page the code doesn't compile.

I've added the required dependencies to the `Cargo.toml` file and the example to `main.rs`

```toml
# Cargo.toml
[dependencies]
gpui = "0.2.2"
gpui-component = "0.5.1"
gpui-component-assets = "0.5.1"
anyhow = "1.0"
```

```rust
// main.rs
use gpui::*;
use gpui_component::{button::*, *};

pub struct HelloWorld;

impl Render for HelloWorld {
    fn render(&mut self, _: &mut Window, _: &mut Context<Self>) -> impl IntoElement {
        div()
            .v_flex()
            .gap_2()
            .size_full()
            .items_center()
            .justify_center()
            .child("Hello, World!")
            .child(
                Button::new("ok")
                    .primary()
                    .label("Let's Go!")
                    .on_click(|_, _, _| println!("Clicked!")),
            )
    }
}

fn main() {
    let app = gpui_platform::application().with_assets(gpui_component_assets::Assets);

    app.run(move |cx| {
        // This must be called before using any GPUI Component features.
        gpui_component::init(cx);

        cx.spawn(async move |cx| {
            cx.open_window(WindowOptions::default(), |window, cx| {
                let view = cx.new(|_| HelloWorld);
                // This first level on the window, should be a Root.
                cx.new(|cx| Root::new(view, window, cx))
            })
            .expect("Failed to open window");
        })
        .detach();
    });
}
```
The app fails to run with the following error:
```
error[E0433]: failed to resolve: use of unresolved module or unlinked crate `gpui_platform`
  --> src/main.rs:25:15
   |
25 |     let app = gpui_platform::application().with_assets(gpui_component_assets::Assets);
   |               ^^^^^^^^^^^^^ use of unresolved module or unlinked crate `gpui_platform`
   |
   = help: if you wanted to use a crate named `gpui_platform`, use `cargo add gpui_platform` to add it to your `Cargo.toml`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `rustic-gui` (bin "rustic-gui") due to 1 previous error
```
Is the example on the website wrong and should I use the git gpui version?

--- a/docs/docs/assets.md
+++ b/docs/docs/assets.md
@@ -17,10 +17,10 @@ The [gpui-component-assets] crate provides a default bundled assets implementati
 
 To use the default bundled assets, you need to add the `gpui-component-assets` crate as a dependency in your `Cargo.toml`:
 
-```toml-vue
+```toml
 [dependencies]
-gpui-component = "{{ VERSION }}"
-gpui-component-assets = "{{ VERSION }}"
+gpui-component = { git = "https://github.com/longbridge/gpui-component" }
+gpui-component-assets = { git = "https://github.com/longbridge/gpui-component" }
 ```
 
 Then we need call the `with_assets` method when creating the GPUI application to register the asset source:

--- a/docs/docs/getting-started.md
+++ b/docs/docs/getting-started.md
@@ -10,12 +10,13 @@ order: -2
 
 Add dependencies to your `Cargo.toml`:
 
-```toml-vue
+```toml
 [dependencies]
-gpui = "{{ GPUI_VERSION }}"
-gpui-component = "{{ VERSION }}"
+gpui = { git = "https://github.com/zed-industries/zed" }
+gpui_platform = { git = "https://github.com/zed-industries/zed" }
+gpui-component = { git = "https://github.com/longbridge/gpui-component" }
 # Optional, for default bundled assets
-gpui-component-assets = "{{ VERSION }}"
+gpui-component-assets = { git = "https://github.com/longbridge/gpui-component" }
 anyhow = "1.0"
 ```
 

--- a/docs/docs/index.md
+++ b/docs/docs/index.md
@@ -27,10 +27,10 @@ GPUI Component is a comprehensive UI component library for building fantastic de
 
 Add `gpui` and `gpui-component` to your `Cargo.toml`:
 
-```toml-vue
+```toml
 [dependencies]
-gpui = "{{ VERSION }}"
-gpui-component = "{{ VERSION }}"
+gpui = { git = "https://github.com/zed-industries/zed" }
+gpui-component = { git = "https://github.com/longbridge/gpui-component" }
 ```
 
 Then create a simple "Hello, World!" application with a button:
@@ -59,9 +59,7 @@ impl Render for HelloWorld {
 }
 
 fn main() {
-    let app = Application::new();
-
-    app.run(move |cx| {
+    gpui_platform::application().run(move |cx| {
         // This must be called before using any GPUI Component features.
         gpui_component::init(cx);
 

--- a/docs/docs/installation.md
+++ b/docs/docs/installation.md
@@ -41,7 +41,7 @@ We use Rust programming language to build the `gpui-component` library. Make sur
 
 To install the `gpui-component` library, you can use Cargo, the Rust package manager. Add the following line to your `Cargo.toml` file under the `[dependencies]` section:
 
-```toml-vue
-gpui = "{{ GPUI_VERSION }}"
-gpui-component = "{{ VERSION }}"
+```toml
+gpui = { git = "https://github.com/zed-industries/zed" }
+gpui-component = { git = "https://github.com/longbridge/gpui-component" }
 ```

--- a/docs/index.md
+++ b/docs/index.md
@@ -28,16 +28,9 @@ GPUI and GPUI Component are under active development, recently GPUI have some ne
 
 The documentation on this site are based on the **Git main branch**, if you use the crates.io version, there may be some differences.
 
-```toml-vue
+```toml
 gpui = { git = "https://github.com/zed-industries/zed" }
 gpui-component = { git = "https://github.com/longbridge/gpui-component" }
-```
-
-If you prefer to use the versions on crates.io. Please visit [docs.rs](https://docs.rs/gpui-component/latest/gpui_component/) to check the API differences.
-
-```toml-vue
-gpui = "{{ GPUI_VERSION }}"
-gpui-component = "{{ VERSION }}"
 ```
 
 ## Hello World
@@ -68,9 +61,7 @@ impl Render for HelloWorld {
 }
 
 fn main() {
-    let app = Application::new();
-
-    app.run(move |cx| {
+    gpui_platform::application().run(move |cx| {
         // This must be called before using any GPUI Component features.
         gpui_component::init(cx);
 

